editor/src/clj/editor/cljfx_form_view.clj

```diff
@@ -1526,52 +1532,57 @@
-                                     (ui/run-command node :open {:resources [value]}))
-                    :show-dialog (fn [dialog-type _]
-                                   (case dialog-type
-                                     :directory-not-in-project
-                                     (dialogs/make-info-dialog
-                                       {:title "Invalid Directory"
-                                        :icon :icon/triangle-error
-                                        :header "The directory must reside within the project."})
-
-                                     :file-not-in-project
-                                     (dialogs/make-info-dialog
-                                       {:title "Invalid File"
-                                        :icon :icon/triangle-error
-                                        :header "The file must reside within the project."})))})
-                 (wrap-force-refresh view-id))}
+                               (form/set-value! ops path value)))
+                      :clear (fn [path _]
+                               (let [ops (:form-ops (g/node-value view-id :form-data))]
+                                 (when (form/can-clear? ops)
+                                   (form/clear-value! ops path))))
+                      :set-ui-state (fn [ui-state _]
+                                      (g/set-property! view-id :ui-state ui-state))
+                      :cancel-edit (fn [x _]
+                                     (cond
+                                       (instance? Cell x)
+                                       (fx/run-later (.cancelEdit ^Cell x))
+
+                                       (instance? ListView x)
+                                       (.edit ^ListView x -1)))
+                      :open-resource (fn [[node value] _]
+                                       (ui/run-command node :open {:resources [value]}))
+                      :show-dialog (fn [dialog-type _]
+                                     (case dialog-type
+                                       :directory-not-in-project
+                                       (dialogs/make-info-dialog
+                                         {:title "Invalid Directory"
+                                          :icon :icon/triangle-error
+                                          :header "The directory must reside within the project."})
+
+                                       :file-not-in-project
+                                       (dialogs/make-info-dialog
+                                         {:title "Invalid File"
+                                          :icon :icon/triangle-error
+                                          :header "The file must reside within the project."})))})
+                   (wrap-force-refresh view-id))}
+              (system/defold-dev?)
+              (assoc :fx.opt/type->lifecycle (requiring-resolve 'cljfx.dev/type->lifecycle)))
```

Comment:
This diff is mostly indentation due to wrapping in `cond->`, here is the actual change: in dev, use `cljfx/dev` lib for better error reporting.